src/Reflex/Dom/SemanticUI/Common.hs

```diff
@@ -65,6 +70,9 @@ instance UiClassText UiColor where
 class UiHasColor a where
   uiSetColor :: UiColor -> a -> a
 
+instance (Reflex t, UiHasColor a) => UiHasColor (Dynamic t a) where
```

Comment:
I hadn't thought that far ahead with respect to Dynamics yet.  :)

I was just trying to think of a way to unify the syntax for different widgets that had similar (but not always exactly the same) sets of operations.  For instance, tables have two levels of compactness: compact and very compact.  But buttons don't.  Also, I don't think all widgets have the full range of sizes from mini to massive.  So I was looking for a system that didn't require both of the following two separate things:

UiTableCompactnessCompact

UiButtonCompactnessCompact